docs/data/date-pickers/getting-started/getting-started.md

```diff
@@ -52,7 +52,7 @@ npm install @mui/material @mui/styled-engine-sc styled-components
 yarn add @mui/material @mui/styled-engine-sc styled-components
 ```
 
-## Code setup
+## Setup your date library adapter
```

Comment:
Whoops, too fast approval. 🙈 
Could you please update the `LocalizationProvider` description containing a link to `#code-setup`? 🙏